2022-05-04 09:37:42,939 INFO  ==============================================================
2022-05-04 09:37:42,940 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
2022-05-04 09:37:42,941 INFO  Dir:   software_management_end_to_end/
2022-05-04 09:37:42,942 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
2022-05-04 09:37:42,943 INFO  ==============================================================
2022-05-04 09:37:42,953 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
2022-05-04 09:37:42,954 INFO  EnvironmentC8y Setup
2022-05-04 09:37:43,516 INFO  Executed systemctl<serv_mapper1>, exit status 3
2022-05-04 09:37:43,622 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
2022-05-04 09:37:43,624 INFO  Delaying execution by 6 seconds
2022-05-04 09:38:03,795 INFO  Executed sudo<tedge_connect_c8y>, exit status 0, duration 14 secs
2022-05-04 09:38:04,361 INFO  Executed sudo<tedge_connect_c8y_test>, exit status 0
2022-05-04 09:38:04,824 INFO  Executed systemctl<serv_mosq2>, exit status 0
2022-05-04 09:38:05,290 INFO  Executed systemctl<serv_mapper3>, exit status 0
2022-05-04 09:38:07,515 INFO  State of current operation: SUCCESSFUL
2022-05-04 09:38:07,739 INFO  Response status: 201
2022-05-04 09:38:07,740 INFO  Response to action: {
    "creationTime": "2022-05-04T09:38:07.693Z",
    "deviceName": "m32sd10d",
    "deviceId": "18360056",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18392314",
    "id": "18392314",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
2022-05-04 09:38:07,741 INFO  Started operation: {'creationTime': '2022-05-04T09:38:07.693Z', 'deviceName': 'm32sd10d', 'deviceId': '18360056', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18392314', 'id': '18392314', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
2022-05-04 09:38:09,966 INFO  State of operation 18392314 : PENDING
2022-05-04 09:38:12,178 INFO  State of operation 18392314 : EXECUTING
2022-05-04 09:38:14,389 INFO  State of operation 18392314 : EXECUTING
2022-05-04 09:38:16,636 INFO  State of operation 18392314 : EXECUTING
2022-05-04 09:38:18,858 INFO  State of operation 18392314 : FAILED
2022-05-04 09:38:18,859 ERROR Stopping as the operation has failed
2022-05-04 09:38:18,860 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
2022-05-04 09:38:18,883 WARN  SystemError: The operation has failed ... blocked 
2022-05-04 09:38:18,884 INFO  
2022-05-04 09:38:18,884 INFO  cleanup:
2022-05-04 09:38:22,793 INFO  Executed sudo<tedge_disconnect_c8y>, exit status 0
2022-05-04 09:38:23,356 INFO  Executed systemctl<serv_mapper5>, exit status 3
2022-05-04 09:38:23,360 INFO  
2022-05-04 09:38:23,361 INFO  Test duration: 40.43 secs
2022-05-04 09:38:23,362 INFO  Test final outcome:  BLOCKED
2022-05-04 09:38:23,363 INFO  Test outcome reason: SystemError: The operation has failed
2022-05-04 09:38:23,364 INFO  
